### PR TITLE
[systemtest] Add asserts for checking java options in EO

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change
- Enhancement / new feature

### Description

Add asserts for checking presence of Java options if we add it to Kafka creation - to EO - topic-operator and user-operator

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=KafkaST#testJvmAndResources test_only

File: systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java [hunk 1]
```diff
@@ -771,6 +783,16 @@ void testJvmAndResources() {
         assertResources(cmdKubeClient().namespace(), pod.get().getMetadata().getName(), "user-operator",
                 "512M", "300m", "256M", "300m");
 
+        String eoPod = eoPods.keySet().toArray()[0].toString();
+        kubeClient().getPod(eoPod).getSpec().getContainers().forEach(container -> {
+            if (!container.getName().equals("tls-sidecar")) {
+                LOGGER.info("Check if -D java options are present in {}", container.getName());
+                String value = container.getEnv().stream().filter(envVar ->
+                        envVar.getName().equals("STRIMZI_JAVA_SYSTEM_PROPERTIES")).collect(Collectors.toList()).get(0).getValue();
+                assertThat(value, is("-Djavax.net.debug=verbose"));
```
Comment (stanlyDoge) [inline systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: Maybe this looks simpler ```String value = container.getEnv().stream().filter(envVar ->
                        envVar.getName().equals("STRIMZI_JAVA_SYSTEM_PROPERTIES")).findFirst().get().getValue();```

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: yeah, good point, thanks :)

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=KafkaST#testJvmAndResources test_only

**Pull request merged**